CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# The @akka/akka-admin-team owns only `.github/settings.yml`
```

Comment:
```suggestion
# The @akka/akka-admin-team owns only `.github/settings.yml` to prevent people
# from adding themselves as admin, see https://github.com/probot/settings#security-implications
```